src/signals/incident-management/containers/IncidentDetail/components/MetaList/index.test.js

```diff
@@ -223,12 +223,13 @@ describe('MetaList', () => {
 
   describe('subcategory', () => {
     const subcategoryLabel = 'Subcategorie (verantwoordelijke afdeling)';
-    const selectedSubcategory = `${parentIncident.category.sub} (${departmentAegCode}, ${departmentAscCode})`;
+    const selectedSubcategory = 'Asbest / accu';
 
     it('should be visible', () => {
-      render(renderWithContext());
+      const { container } = render(renderWithContext());
+
       expect(screen.getByText(subcategoryLabel)).toBeInTheDocument();
-      expect(screen.getByText(selectedSubcategory)).toBeInTheDocument();
+      expect(screen.getByText(selectedSubcategory, { exact: false })).toBeInTheDocument();
```

Comment:
Feels cleaner to make the selectedSubcategory string a Regex instead of using the exact property. That's also the recommended approach explained [here](https://testing-library.com/docs/dom-testing-library/api-queries/#precision)